base/Dockerfile

```diff
@@ -28,6 +28,16 @@ RUN sed -i 's#/archive.ubuntu.com#/au.archive.ubuntu.com#g' /etc/apt/sources.lis
         zsh \
         nano \
         emacs \
+        screen \
+        build-essential \
+        libssl-dev \
+        libffi-dev \
+        libreadline-dev \
+        libncurses5-dev \
+        libbz2-dev \
+        liblzma-dev \
+        zlib1g-dev \
+        libsqlite3-dev \
```

Comment:
Adding these because `pyenv install <version>` would fail without these dependencies.

```
Installing Python-3.8.15...
WARNING: The Python bz2 extension was not compiled. Missing the bzip2 lib?
WARNING: The Python curses extension was not compiled. Missing the ncurses lib?
WARNING: The Python ctypes extension was not compiled. Missing the libffi lib?
WARNING: The Python readline extension was not compiled. Missing the GNU readline lib?
ERROR: The Python ssl extension was not compiled. Missing the OpenSSL lib?
```

Reply:
Wonder if we should comment on the block to explain the "upstream" reason for all these packages, so we don't regress support in the future.

Also wonder if there's some other package that hits all these already?

Reply:
Have added a comment now. And have filtered any redundant entries from the install list.